ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoAttachments.vue

```diff
@@ -378,14 +413,21 @@ export default {
       this.showDocumentSelector = !this.showDocumentSelector;
       this.drawerTitle = this.showDocumentSelector? `${this.$t('attachments.drawer.existingUploads')}` : `${this.$t('attachments.drawer.header')}`;
       if (!this.showDocumentSelector){
-        this.showDestinationFolder = false ;
+        this.showDestinationFolder = false;
+        this.showDestinationFolderForFile = false;
       }
     },
     toggleSelectDestinationFolder(){
       this.showDestinationFolder = true ;
       this.showDocumentSelector = !this.showDocumentSelector;
       this.drawerTitle = this.showDocumentSelector? `${this.$t('attachments.drawer.destination.folder')}` : `${this.$t('attachments.drawer.header')}`;
     },
+    toggleSelectDestinationFolderForFile(file){
```

Comment:
```suggestion
    openSelectDestinationFolderForFile(file){
```
"toggle" is for open and close, whereas we only open here.